src/fortrancode.l

```diff
@@ -932,14 +932,14 @@ INTRINSIC_DERIVED_TYPE          (C_FUNPTR|C_PTR|EVENT_TYPE|LOCK_TYPE|TEAM_TYPE)
                                                             codifyLines(yyscanner,yytext);
                                                             endFontClass(yyscanner);
                                                         }
-
+ /*
 <*>^[Cc*].*                                             {   // normal comment
                                                             if(! yyextra->isFixedForm) YY_FTN_REJECT;
                                                             startFontClass(yyscanner,"fortran_comment");
                                                             codifyLines(yyscanner,yytext);
                                                             endFontClass(yyscanner);
                                                         }
-
+ */
```

Comment:
The old style comment is still standard Fortran.

Reply:
What is the fix for this?

Reply:
Most  likely the not inclusion of the lines 935 and 942, see also https://github.com/doxygen/doxygen/pull/8774#issuecomment-918355501

Reply:
The F77 comment style can be revived by removing the lines 935 and 942 as Albert says. However, adding it will mess up with Fortran90 source code lines that start with C or c. One option is to limit this comment style to .f and .for file extensions since this F77 comment style is not recognized and is illegal in the modern free-format Fortran files with extension .f90.
Another option is to modify the rule to better recognize the F77 comment-style where they appear without interfering with the syntax highlighting of free-format f90 source files.

Reply:
@shahmoradi 
Can you give an example where normally the free formatted code is messed up due to this rule?
I can imagine that this is happening in the code coloring as I see in the lexical scanner (to determine the syntax) there is an attempt to convert from fixed formatted code to free formatted code, for source code output this conversion cannot be done and something different has to be done and is done as well but maybe not everywhere.

Reply:
Yes, I do not have a minimal reproducible sample right now since I removed the F77 style comment rule in the fork longtime ago. But I remember this happened frequently in code sections within the documentation where Doxygen highlighted a `call xxx` statement starting at the first column as a comment line.